deploy/crds/push_v1alpha1_androidvariant_crd.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: androidvariants.push.aerogear.org
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.variantId
+    name: VariantId
+    type: string
+  - JSONPath: .status.ready
+    name: Ready
+    type: string
+  group: push.aerogear.org
+  names:
+    kind: AndroidVariant
+    listKind: AndroidVariantList
+    plural: androidvariants
+    singular: androidvariant
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            description:
+              description: Description is human friendly description for the variant.
+              type: string
+            senderId:
+              description: SenderId is the "Google Project Number" from the API Console.
+                It is *not* needed for sending push messages, but it is a convenience
+                to "see" it on the Admin UI as well, since the Android applications
+                require it (called Sender ID there). That way all information is stored
+                on the same object.
+              type: string
+            serverKey:
+              description: ServerKey is key from the Firebase Console of a project
+                which has been enabled for FCM.
+              type: string
+          required:
+          - serverKey
+          - senderId
+          type: object
+        status:
+          properties:
+            ready:
+              type: boolean
+            variantId:
+              type: string
+          required:
+          - ready
+          type: object
+  version: v1alpha1
+  versions:
+  - name: v1alpha1
```

Comment:
I'm not sure if this question is valid, asking anyway! :) Did the older ups-config-operator have CRDs, and if so was there one called AndroidVariant?

Reply:
it was using the binding mechanism with secrets. No CRDs!